### DISCUSSION TITLE
How to pass an svg file along with esm and css files in anywidget ?

### DISCUSSION THREAD

**Discussion by anmolbhatia05** [manzt/anywidget#655] [JavaScript]
hello, i am trying to create this graph editing widget. 

my anywidget class looks like this:
```
class GraphEditingWidget(anywidget.AnyWidget):
    """Something."""

    _esm: pathlib.Path = pathlib.Path(__file__).parent / "static" / "widget.js"
    _css: pathlib.Path = pathlib.Path(__file__).parent / "static" / "widget.css"
    data = traitlets.Unicode(default_value="{}").tag(sync=True)
    height = traitlets.Int(default_value=600).tag(sync=True)
    is_dag = traitlets.Bool(default_value=True).tag(sync=True)
```

my widget.js is based on [cytoscape](https://js.cytoscape.org/) and the node editing capability is from an extension called [cytoscape node-editing](https://github.com/iVis-at-Bilkent/cytoscape.js-node-editing?tab=readme-ov-file).

for the node editing to work the extension needs access to this svg file called `resizeCue.svg`., which is passed as a relative file path attribute in the extension, like so - 
```
resizeToContentCueImage: 'resizeCue.svg',
```

how do i pass this svg file alongwith the esm and css in anywidget class mentioned above ? 

maybe some bundling magic, i am using esbuild as a bundler already

**Comment by psychemedia** [comment]
I haven't passed svg, but I do pass html from a file, loading it into the js widget as `import html from "./myhtml.html";` and in the `package.json` `scripts.build` element calling `esbuild ...  --loader:.html=text`
